src/dune_rules/stanzas/buildable.ml

```diff
@@ -43,6 +44,10 @@ let decode_modules = Modules_settings.decode
 let decode_lint = field "lint" Lint.decode ~default:Lint.default
 let decode_allow_overlapping = field_b "allow_overlapping_dependencies"
 
+let decode_allow_unused_libraries =
+  field "allow_unused_libraries" (repeat (located Lib_name.decode)) ~default:[]
```

Comment:
A slightly different semantics would have been to have a field `unused_libraries` which would error out if they were used, but I guess that would need a slightly different implementation. In practice, I imagine that someone using this feature would *know* whether the library is supposed to be unused or not. What do you think?

Reply:
Yeah, I could make it work like that. In fact, that's how i made it work internally. I don't remember why I changed it for the worse here. But something else is bothering me about this field - the duplication in the library list that it causes. I'm just going to guard this feature behind a feature flag for now. We can change the implementation of this alias without additional versioning since it's not attached to `runtest` at the moment.